NEWS

```diff
@@ -32,8 +32,9 @@ When a gap character is defined, Bio.Seq will now translate gap codons
 is inferred from the Seq object's alphabet, but it can also be passed as an
 argument to the translate method.
 
-The new NCBI genetic code table 25, Candidate Division SR1 and Gracilibacteria,
-has been added to Bio.Data (and the translation functionality).
+Three new datasets have been added to Bio.Data (and the translation
```

Comment:
Its one dataset (table 25), covering both Candidate Division SR1 and Gracilibacteria:

https://www.ncbi.nlm.nih.gov/Taxonomy/Utils/wprintgc.cgi#SG25